### DISCUSSION TITLE
[Enhancement] Replace px with rem

### DISCUSSION THREAD

**Discussion by jpmaligne** [element-plus/element-plus#13938] [Feature Request]
Feature type: **`Enhancement`**

## Details

- Enhanced component name: `All`

## I think Element-plus should transit from px to rem in style declaration

Is there a way to use element-plus in an application willing to use rem and boost accessibility ?

### Features
1. Main benefit is to increase accessibility
2. Today, in an app using element-plus, transiting to rem is impossible due to the fact that element-plus won't scale with the rest of the app. (unless I'm missing a point)

### Living examples
Bootstrap is fully rem compliant https://github.com/twbs/bootstrap/blob/main/scss/_variables.scss

**Comment by oliver139** [comment]
Already about a year, and I think this is compulsory

**Comment by cvpv** [comment]
我该如何自定义主题，是得element plus完全基于rem单位工作，以便更好的适配tailwind css等其他css工具

How can I customize the theme so that Element Plus works entirely based on the rem unit for better compatibility with other CSS tools like Tailwind CSS?

**Reply by rzzf** [reply]
你可以使用[postcss-pxtorem](https://github.com/cuth/postcss-pxtorem)插件

**Reply by cvpv** [reply]
> 你可以使用[postcss-pxtorem](https://github.com/cuth/postcss-pxtorem)插件

好的，非常感谢。

但是实际上我的项目并没有tailwind，而是使用unocss ，并没有引入postcss。我目前的想法是完全定制一套自己的主题，不引入官方的scss。
